webapp/src/main/webapp/src/app/app.routes.ts

```diff
@@ -13,13 +14,17 @@ export const routes: Routes = [
         path: '',
         pathMatch: 'full',
         component: HomeComponent
-      },
-      {
+      }, {
         path: 'groups',
-        pathMatch: 'full',
         resolve: { filterOptions: GroupFilterOptionsResolve },
-        data: { breadcrumb: { translate: 'labels.groups.title' } },
-        component: GroupsComponent
+        data: { permissions: [ 'GROUP_READ' ]},
+        canActivate: [ AuthorizeCanActivate ],
+        children: [ {
+          path: 'filterBy',
```

Comment:
@wtritch, this is how I solved the issue we were having in Reporting in which only changing the params was causing the parent route to re-resolve it's params.  This was happening because were were using a child path with `''`.  But if I give a name to the child path `'filterBy'` then it works as we expect it to.